src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -11001,6 +11113,64 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/ActionBodyExecTemplate'
+  RuleSetPaging:
+    type: object
+    required:
+      - list
+    properties:
+      list:
+        type: object
+        required:
+          - pagination
+          - entries
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/RuleSetEntry'
+  RuleSetEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/RuleSet'
+  RuleSet:
+    type: object
+    required:
+      - id
+      - ownedBy
+    properties:
```

Comment:
I think we should have an extra property `shared` which says whether the rule set is shared or not. This way a user can tell that a rule set is shared even if they don't have access to all the folder ids.

Reply:
That answers my other comment. I will do that!